hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/OmUtils.java

```diff
@@ -321,6 +321,7 @@ public static boolean isReadOnly(
     case SnapshotPurge:
     case RecoverLease:
     case SetTimes:
+    case SnapshotUpdateSize:
```

Comment:
This can be generalized/merged with other operations that updates `SnapshotInfo`:

```suggestion
    case SetSnapshotProperty:
```